addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.trakt" name="Trakt" version="3.2.4" provider-name="Trakt.tv, Razzeee">
+<addon id="script.trakt" name="Trakt" version="3.2.4b" provider-name="Trakt.tv, Razzeee">
```

Comment:
Can you remove this for now? I like to keep code and the version bumps separated :)

Reply:
Done